wolfcrypt/test/test.c

```diff
@@ -8587,13 +8638,16 @@ int aesgcm_test(void)
     #ifndef BENCH_AESGCM_LARGE
         #define BENCH_AESGCM_LARGE 1024
     #endif
-    byte large_input[BENCH_AESGCM_LARGE];
-    byte large_output[BENCH_AESGCM_LARGE + AES_BLOCK_SIZE];
-    byte large_outdec[BENCH_AESGCM_LARGE];
+    byte *large_input = (byte *)XMALLOC(BENCH_AESGCM_LARGE, NULL, DYNAMIC_TYPE_TMP_BUFFER);
```

Comment:
You might consider not running this test large AES test in the `BENCH_EMBEDDED` case and setting up LINUXKM to use `BENCH_EMBEDDED`.

Reply:
it's actually already gated out when `-DBENCH_EMBEDDED`.  but in any case, it seems to be fine for linuxkm.  negligible runtime, and
```
AES-GCM  test passed!
relative stack used = 3784
AES-CCM  test passed!
relative stack used = 3912
```
modest stack.

linuxkm doesn't map neatly to `BENCH_EMBEDDED`, because it's really just stack that's invariably at a super-premium.  otherwise, typically a large heap, rich instruction set, and fast cores.

still really psyched that the AESNI asm is working.  in kernel configure trim, we get
```
AES-256-GCM-enc   3415 MB took 1.001 seconds, 3412.748 MB/s Cycles per byte =   0.95
AES-256-GCM-dec   3395 MB took 1.000 seconds, 3394.705 MB/s Cycles per byte =   0.95
```
the full monty!